packages/neon-core/src/tx/transaction/IssueTransaction.ts

```diff
@@ -0,0 +1,51 @@
+import { ASSET_ID, DEFAULT_SYSFEE } from "../../consts";
+import { StringStream } from "../../u";
+import { BaseTransaction, TransactionLike } from "./BaseTransaction";
+import TransactionType from "./TransactionType";
+
+export class IssueTransaction extends BaseTransaction {
+  public static deserializeExclusive(
+    ss: StringStream,
+    tx: Partial<TransactionLike>
+  ): Partial<TransactionLike> {
+    return {};
+  }
+
+  public readonly type: TransactionType = TransactionType.IssueTransaction;
+
+  public constructor(obj: Partial<TransactionLike> = {}) {
+    super(obj);
```

Comment:
looks like the latest version is 1 so we should default to setting the version to 1.

Reply:
@snowypowers , are you sure? Which is the latest version that you are looking at?

Reply:
im looking at neo source code. There is indication of the existance of version 1 and I would assume that we should be using the latest version template if possible. That said, I believe this is a deprecated transaction type so might not be relevant.

Reply:
Actually the issue transaction are not deprecated. If I look at the [C# neo code](https://github.com/neo-project/neo/blob/master/neo/Network/P2P/Payloads/IssueTransaction.cs#L30) form the master branch I wouldn't be sure that the default version should be set to one: 
```C#
 protected override void DeserializeExclusiveData(BinaryReader reader)
        {
            if (Version > 1) throw new FormatException();
        }
```
The valid versions are either 0 or 1. But so far I have found only about 10 issue transactions and all of them have version 0.  I am currently dumping deserialize json transactions form the python bootstrap Mainet chain into mongo db. From there I will be able to make quick queries but it will take a lot of time to dump everything. Do you still want me to set the default version to one or shall we analyse further?

Reply:
Lets see if there is anything with version 1. As far as I know, its not marked deprecated but in reality it is not used because it is used together with the one of the deprecated ones.

Reply:
The dumping process has covered so far about 10'500'000 transactions. Grouping the transaction by type we get to the following statistics:
```
{ "_id" : "PublishTransaction", "count" : 1 }
{ "_id" : "IssueTransaction", "count" : 6 }
{ "_id" : "EnrollmentTransaction", "count" : 6 }
{ "_id" : "RegisterTransaction", "count" : 13 }
{ "_id" : "MinerTransaction", "count" : 1448857 }
{ "_id" : "ClaimTransaction", "count" : 1914964 }
{ "_id" : "InvocationTransaction", "count" : 3427266 }
{ "_id" : "StateTransaction", "count" : 6 }
{ "_id" : "ContractTransaction", "count" : 3608881 }
```
So far all the issue transactions have version =0. I will wait that the dumping process will stop hoefully in 2-3 days and cross check again whether we have any issue transaction with version 1.

Reply:
After analyzing more 19'000'000  transactions we found only 16 `IssueTransaction` and all of them have version 0. Considering that mainet has about 22'000'000, I would says that it is ok to set the default version of the IssueTransaction to 0. Did you agree?

Reply:
ok v=0.